src/wallet/swapcoin.rs

```diff
@@ -32,7 +32,18 @@ use crate::protocol::{
 
 use super::WalletError;
 
-/// Represents an incoming swapcoin.
+/// Defines an incoming swapcoin, which can either be currently active or successfully completed.
+///
+/// ### NOTE:
+/// The term `Incoming` may imply an ongoing process.
+/// However, in the context of this protocol, it specifically refers to a coin
+/// that was initially received by a participant during a swap transaction.
```

Comment:
Better not to use "may". There is no ambiguity about what they mean. 
```suggestion
/// The term `Incoming` imply an Incoming Coin from a swap.
/// This can be for either a Taker or a Maker, depending on their position in the swap route.
/// This refers to coins that have been "received" by the party, i.e. the party holds the hash lock side of the contract.
```